screens/CommentModal.tsx

```diff
@@ -209,8 +205,15 @@ export const CommentModal = ({ route }: RootStackScreenProps<"Comment">) => {
                     <View style={styles.comment}>
                       <Image source={{ uri: c.image }} style={styles.image} />
                       <View style={styles.textWrapper}>
-                        <Text>{c.author}</Text>
-                        <Text>{c?.text}</Text>
```

Comment:
The comment text is removed. Please add it back :) 

<img width="335" alt="Screenshot 2023-01-20 at 10 46 03" src="https://user-images.githubusercontent.com/80100462/213665180-783f66fd-2355-40ab-89cd-d0fcacc2b0a1.png">